### PR TITLE
initialize readme

### PR DESCRIPTION
**Pull request by kylebuttner**

### CODE REVIEW

**Comment by mechacoach** [comment]
You had your airport_challenge reviewed by **Yiorgos Joseph Makridakis**

### The good points are:

* `context` or `describe` blocks are used (where appropriate) to group tests
* `before` blocks are used where appropriate
* There are a minimal number of untested paths in the code
* Unit tests all have single expectations
* The code follows Ruby naming conventions, Readability is supported by consistent naming
* Guard clauses are used where appropriate, Booleans are returned implicitly where appropriate, removeallcommented-outcode: There is no commented-out code in the submission
* Code avoids unnecessary public methods
* Classes have single responsibilities, Each methods has a single responsibility
* Code avoids strings where it should use symbols
* Command/Query naming is not an issue in the code
* The solution does not have any redundant code

### You should consider the following improvements:

* README should be updated
* Installation Instructions should be in the README
* Use named subject with `described_class`
* Avoid Vacuous tests
* Handle randomness in tests
* Get rid of redundant `respond_to` expectations
* Remove all commented-out code
* Avoid magic numbers (e.g. on capacity)
* Improper use of attr_*

### Additional comments:
Quite impressive. Had issues stubbing random behaviour of weather class. Didn't reach capacity part. Seems to know whats going on.

see https://github.com/makersacademy/airport_challenge/blob/master/docs/review.md for more details

File: lib/airport.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    plane.depart
+    planes.delete(plane)
+  end
+
+  def present?(plane)
+    planes.include? plane
+  end
+
+  def is_stormy?
+    Weather.new.stormy?
```
Comment (danldb) [inline lib/airport.rb]: you could use dependency injection here to remove the coupling between Weather and Airport

File: lib/airport.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  def at_capacity?
```
Comment (danldb) [inline lib/airport.rb]: this should really be private

File: spec/airport_spec.rb [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  #error messages
+  not_flying_error = "This plane is not currently flying."
+  not_at_airport_error = "This plane is not at this airport."
+  weather_error = "This plane cannot depart due to adverse weather conditions"
+  capacity_error = "This airport is currently at capcity."
+
+  context 'when weather is not stormy' do
+    let(:weather) { double(:weather) }
+    before(:each) do
+      allow(airport).to receive(:is_stormy?) { false }
```
Comment (danldb) [inline spec/airport_spec.rb]: don't stub your subject

**Pull request closed without merging**